tests/bluemira/radiation_transport/test_advective_transport.py

```diff
@@ -181,3 +181,22 @@ def setup_class(cls):
     def test_recursion(self):
         assert np.isclose(np.max(self.hf), 86.194, rtol=1e-2)
         assert np.isclose(np.sum(self.hf), 830.6, rtol=1e-2)
+
+    def test_analyse_DN(self, caplog):
```

Comment:
I don't like this test, because it's not testing the API, and is in fact hacking the solver around to test some underlying stuff.